tests/integration/backward_compatible/cluster_test.py

```diff
@@ -213,6 +213,35 @@ def assertion():
         self.assertEqual(1, len(members))
         self.assertEqual(member.uuid, str(members[0].uuid))
 
+    def test_when_member_started_with_the_same_address(self):
+        mark_client_version_at_least(self, "4.2")
```

Comment:
I think I forgot to add this check in Node.js client 

https://github.com/hazelcast/hazelcast-nodejs-client/pull/896/files#diff-0bf3994a78093774f530d1d4e4521bc0da29c41e1a476312f62735f136065523R84

version check is necessary since we fixed this in 4.2 right?

I guess my test in Nodejs will fail in the next backward compat tests when a new imdg is about to be released. We need to add the version check to Nodejs too, right?

Reply:
Yes, we probably need to add this to the Node.js client as well